lib/ExpensiMark.js

```diff
@@ -157,6 +157,23 @@ export default class ExpensiMark {
                 replacement: '<br>',
             },
         ];
+
+        /**
+         * The list of regex replacements to do on a HTML comment for converting it to markdown.
+         *
+         * @type {Object[]}
+         */
+        this.htmlToMarkdownRules = [
+            {
+                name: 'newline',
+
+                // Replaces open and closing <br><br/> tags with a single <br/>
+                pre: inputString => inputString.replace('<br><br/>', '<br/>'),
+                regex: /<br\s*[\\/]?>/gi,
```

Comment:
I have a few questions regarding our regex pattern: 
 - Why are we attempting to detect `<br\>`? 
 - What is the purpose of using the case insensitive flag?

Reply:
Hi, 
1. we can remove `\` regex
2. This is because HTML is a case insensitive language 

I am thinking of adding `<br></br>` as well in `pre`

Reply:
Ok great! Thx. This should work nicely, idk if the `pre` is necessary.

Reply:
Initially having a regex worked, but later when I was writing some tests and figuring out edge cases. It became necessary to add pre.

Usually, some sanitisers work by appending a closing tag for the uncloses tags `<br>`. That is the reason behind adding pre. Or you can say my addiction to defensive programming 😅

Reply:
I am also accounting for both HTML style and react style of closing